### PR TITLE
russian translation (which is hopefully correct?)

### PR DESCRIPTION
**Pull request by oatmealine**
lots of words didnt really have a correct or accurate russian translation so i just used the english ver (such as debug, checksum, etc)

### CODE REVIEW

File: Celeste.Mod.mm/Content/Dialog/Russian.txt [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+# Title Screen
+	MENU_TITLESCREEN_RESTART_VANILLA= Перезагружаюсь в orig/Celeste.exe
+    
+# Extra Key Mapping
+    KEY_CONFIG_ADDING= НАЖМИТЕ ДОПОЛНИТЕЛЬНУЮ КЛАВИШУ ДЛЯ
+    KEY_CONFIG_ADDITION_HINT= Нажмите SHIFT + CONFIRM что-бы добавить дополнительную клавишу
+
+# Mod Options
+	MODOPTIONS_TITLE= 										EVEREST
+	MODOPTIONS_NEEDSRELAUNCH=								Необхадима перезагрузка что-бы иметь эффект
```
Comment (0x0ade) [inline Celeste.Mod.mm/Content/Dialog/Russian.txt:45]:
```suggestion
	MODOPTIONS_NEEDSRELAUNCH=								Не обхадима перезагрузка чтобы иметь эффект
```

File: Celeste.Mod.mm/Content/Dialog/Russian.txt [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	POSTCARD_LEVELLOADFAILED= 	{big}Простите!{/big}{n}Нам не удалось загрузить{#ff1144}((sid)){#}{n}Пожайлуста, отправте ваш {#44adf7}log.txt{#}{n} создателю карты.
+	POSTCARD_LEVELNOSPAWN= 		{big}Упс!{/big}{n}Кто-то (или что-то) поставило ваше{n}{#44adf7}место начала{#} не на место!{n}Проверте, есть ли в вашей карте{n}объект {#44adf7}Player{#}.
+
+# Main Menu
+	MENU_TITLETOUCH= 		НАЖМИТЕ
+	MENU_MAPLIST= 			Список Карт
+	MENU_MODOPTIONS= 		Настройки Модов
+	MENU_PAUSE_MODOPTIONS= 	Настройки Модов
+	
+	MENU_MODOPTIONS_ONE_MOD_FAILEDTOLOAD=		Неудалось загрузить 1 мод
```
Comment (0x0ade) [inline Celeste.Mod.mm/Content/Dialog/Russian.txt:32]:
```suggestion
	MENU_MODOPTIONS_ONE_MOD_FAILEDTOLOAD=		не удалось загрузить 1 мод
```

File: Celeste.Mod.mm/Content/Dialog/Russian.txt [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+	MENU_MODOPTIONS_MULTIPLE_MODS_FAILEDTOLOAD=	Неудалось загрузить {0} мода
```
Comment (0x0ade) [inline Celeste.Mod.mm/Content/Dialog/Russian.txt:33]:
```suggestion
	MENU_MODOPTIONS_MULTIPLE_MODS_FAILEDTOLOAD=	Не удалось загрузить {0} мода
```

File: Celeste.Mod.mm/Content/Dialog/Russian.txt [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+	MODOPTIONS_COREMODULE_NONTHREADEDGL=					GL без thread-ов # not really a way to translate that correctly
+	MODOPTIONS_COREMODULE_MODUPDATES= 						Проверить обновления модов
+	
+	MODOPTIONS_COREMODULE_RECRAWL= 							~Перезагрузить Всё~
+
+	MODOPTIONS_COREMODULE_SOUNDTEST= 						Тест Звука
+
+	MODOPTIONS_COREMODULE_OOBE= 							Переделать начальную загрузку
+
+	MODOPTIONS_COREMODULE_NOTLOADED_A= 						Неудалось загрузить некоторые моды.
```
Comment (0x0ade) [inline Celeste.Mod.mm/Content/Dialog/Russian.txt:81]:
```suggestion
	MODOPTIONS_COREMODULE_NOTLOADED_A= 						Не удалось загрузить некоторые моды.
```

**Comment by oatmealine** [comment]
there, that shouldve fixed everything

**Pull request merged**